coalib/bearlib/languages/definitions/ActionScript3.py

```diff
@@ -0,0 +1,6 @@
+from coalib.bearlib.languages.Language import Language
+
+
+@Language
+class ActionScript3:
```

Comment:
yep, CC @userzimmermann one big omissioln in the Languages is that you cannot *yet* define different settings for different versions although the whole design is obviously built for it. We're not sure yet which syntax to use to make it intuitive.